Automattic/jetpack
# Add endpoint to fetch site URLs

#### Changes proposed in this Pull Request:

* This endpoint adds a REST API endpoint for fetching site URLs by modified date (descending);
* Only posts from public post types (except built-in ones like attachments) are queried;
* Front page and posts page (where applicable) are also present each time.

#### Other information:


#### Jetpack product discussion


pc9hqz-1of-p2

#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Checkout branch;
* Allow access to endpoint by commenting out the permission requirements on [this line](https://github.com/Automattic/jetpack/pull/27497/files#diff-18d6501ecb2a9a0f0672a577d3760fda782150592c3f056efcad270408142c4dR27);
* Open `site_url/wp-json/jetpack-boost/v1/list-site-urls`;
* You should see a list of page URLs and their modified date.
